webvtt.go

```diff
@@ -188,10 +192,17 @@ func ReadFromWebVTT(i io.Reader) (o *Subtitles, err error) {
 				item.Lines = append(item.Lines, Line{Items: []LineItem{{Text: line}}})
 			default:
 				// This is the ID
-				// TODO Do something with the id
+				// o.Items is empty initially - implementing to save indexe(s) later.
+				if index, err := strconv.Atoi(line); err == nil {
+					indexContent = append(indexContent, index)
+				}
 			}
 		}
 	}
+	// Attach indexContent to the correct position.
```

Comment:
No need for this logic. Instead we can use the value of `index` [here](https://github.com/asticode/go-astisub/blob/master/webvtt.go#L119) (keep the alphabetical order) : 

```go
item = &Item{
				Comments:    comments,
				Index:       index,
				InlineStyle: &StyleAttributes{},
			}
``` 

Don't forget to reset the index just below in case index has not been provided for a specific item :

```go
// Reset index
index = 0
```

Reply:
fixed with https://github.com/asticode/go-astisub/pull/22/commits/489f53adc2376337082e86637e00f39a74a929b2